packages/schematics/src/tslint/nxEnforceModuleBoundariesRule.spec.ts

```diff
@@ -6,7 +6,7 @@ import { Rule } from './nxEnforceModuleBoundariesRule';
 describe('Enforce Module Boundaries', () => {
   it('should not error when everything is in order', () => {
     const failures = runRule(
-      { allow: ['@mycompany/mylib/deep'] },
+      { allow: ['@mycompany/mylib/deep'], lazyLoad: ['myliblazy'] },
```

Comment:
Why is this addition required? It doesn't look like we reference myliblazy anywhere in tests.